packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesConfiguration.java

```diff
@@ -138,6 +152,17 @@ public UpdatesConfiguration loadValuesFromMap(Map<String, Object> map) {
     return this;
   }
 
+  private void maybeSetDefaultScopeKey() {
+    // set updateUrl as the default value if none is provided
+    if (mScopeKey == null) {
+      if (mUpdateUrl != null) {
+        mScopeKey = mUpdateUrl.getScheme() + "://" + mUpdateUrl.getHost();
```

Comment:
Need to get the port too and include it if it's not null and if it's not the default port for the scheme: https://url.spec.whatwg.org/#url-miscellaneous